### PR TITLE
Removed depreciation message with strtoupper

### PR DESCRIPTION
**Pull request by krytenuk**
Updated composer.json

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "shardj/zf1-future",
+    "name": "krytenuk/zf1-future",
```
Comment (PHPGangsta) [inline composer.json:2]: Mistake?

Comment (krytenuk) [reply composer.json:2]: Possibly, sorry I'm new to git and I needed to change this to include it in my project.

Comment (krytenuk) [reply composer.json]: Have reverted change

Comment (krytenuk) [reply composer.json]: Sorry new to git

**Review by randriese — APPROVED** [review]
lgtm

**Comment by Shardj** [comment]
Sorry, just merged another person's deprecation fixes and one of them happened to be fixing the same line as one of yours so there's now a merge conflict.

I do think their cast to string is the way strtoupper currently handles null input. You could also do `strtoupper($input >> "")` for the same results. But returning null if the input is null is a change in behaviour.

**Comment by krytenuk** [comment]
Resolved conflict and improved code by type casting

**Pull request merged**